### PR TITLE
RHDEVDOCS-5389: Correct the formatting of sample commands in Pipeline…

### PR DESCRIPTION
**Pull request by eromanova97**
• **Aligned team**: Dev Tools
• **OCP version for cherry-picking**: enterprise-4.11 and later
• **JIRA issue**: [RHDEVDOCS-5389](https://issues.redhat.com/browse/RHDEVDOCS-5389)
• **Preview page**: [Using Pipelines as Code with a GitHub App](https://61415--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/using-pipelines-as-code.html#using-pipelines-as-code-with-a-github-app_using-pipelines-as-code)
• **SME Review**: **Not needed**
• **QE review**: **Not needed**
• **Peer-review**: Completed by @Srivaralakshmi

### CODE REVIEW

File: modules/op-using-pipelines-as-code-with-a-github-app.adoc [hunk 1]
```diff
@@ -39,7 +39,13 @@ To create and configure a GitHub App manually for {pac}, perform the following s
...
 * **Homepage URL**: OpenShift Console URL 
-* **Webhook URL**: The {pac} route or ingress URL. You can find it by running the command `echo https://$(oc get route -n openshift-pipelines pipelines-as-code-controller -o jsonpath='{.spec.host}')`.
+* **Webhook URL**: The {pac} route or ingress URL. You can find it by running the following command:
++
+[source,terminal]
+----
+$ echo https://$(oc get route -n openshift-pipelines pipelines-as-code-controller -o jsonpath='{.spec.host}')
+----
+
 * **Webhook secret**: An arbitrary secret. You can generate a secret by executing the command `openssl rand -hex 20`.
```
Comment (Srivaralakshmi) [inline modules/op-using-pipelines-as-code-with-a-github-app.adoc:49]: How about this? Do we need to put the command `openssl rand -hex 20` like the previous one?

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@ekristova PTAL at my query.

File: modules/op-using-pipelines-as-code-with-a-github-app.adoc (hunk 1 above)
Comment (ekristova) [reply modules/op-using-pipelines-as-code-with-a-github-app.adoc]: @Srivaralakshmi You are right. Thank you for noticing!

**Review by Srivaralakshmi — APPROVED** [review]
@ekristova Thanks for the changes. Approved!

**Pull request merged**